src/main/groovy/com/readytalk/gradle/plugins/CiLifecyclePlugin.groovy

```diff
@@ -99,10 +99,66 @@ class CiLifecyclePlugin implements Plugin<Project>, PluginUtils {
         }
       }
 
+      withPluginId('com.github.kt3k.coveralls') {
+        tasks.ci.dependsOn tasks.coveralls
+        tasks.coveralls { cover ->
+          onlyIf { infoExt.isCI.toBoolean() }
+        }
+      }
+
+      //Open source plugin project conventions
+      withPlugins(['nu.studer.plugindev', 'com.jfrog.artifactory', 'com.jfrog.bintray']) {
```

Comment:
Only gradle plugin projects have a reason to include the plugindev plugin, making this a reasonable set of triggers for enabling gradle plugin project conventions for publishing.